pcre.yaml

```diff
@@ -0,0 +1,46 @@
+# Generated from https://git.alpinelinux.org/aports/plain/main/pcre/APKBUILD
+package:
+    name: pcre
+    version: "8.45"
+    epoch: 0
+    description: Perl-compatible regular expression library
+    target-architecture:
+        - all
+    copyright:
+        - paths:
+            - '*'
+          attestation: TODO
+          license: BSD-3-Clause
+environment:
+    contents:
+        packages:
+            - busybox
+            - ca-certificates-bundle
+            - build-base
+            - automake
+            - autoconf
+            - m4
+            - libtool
+            - autoconf
+            - automake
+pipeline:
+    - uses: fetch
+      with:
+        expected-sha256: 4dae6fdcd2bb0bb6c37b5f97c33c2be954da743985369cddac3546e3218bffb8
+        uri: https://downloads.sourceforge.net/project/pcre/pcre/${{package.version}}/pcre-${{package.version}}.tar.bz2
+    - uses: autoconf/configure
```

Comment:
I noticed that alpine uses some configure options
```
    with:
      opts: |
        --prefix=/usr \
            --enable-jit \
            --enable-utf8 \
            --enable-unicode-properties \
            --enable-pcre8 \
            --enable-pcre16 \
            --enable-pcre32 \
            --with-match-limit-recursion=8192
```
Though I'm not sure what these mean or if we need these + the related subpackages

Reply:
Added!